### PR TITLE
Add model snap

### PR DESCRIPTION
**Pull request by artivis**

### CODE REVIEW

File: turtlebot3c-model-snap/snapcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    override-build: |
+      craftctl default
+
+      mkdir -p "${CRAFT_PART_BUILD}/xacro"
+
+      set +u
+      source /opt/ros/humble/setup.bash
+      set -u
+
+      #rosdep install --from-path "${CRAFT_PART_SRC_WORK}/nav2_minimal_tb3_sim" -y -i
```
Comment (Guillaumebeuzeboc) [inline turtlebot3c-model-snap/snapcraft.yaml:58]: To remove?

File: turtlebot3c-model-snap/snapcraft.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  urdf-packager:
+    plugin: python
+    source: https://github.com/ubuntu-robotics/urdf-packager.git
+    source-branch: feat/initial-upload
+    override-prime: echo "prime nothing"
+
+  turtlebot3c-model:
+    after: [urdf-packager]
+    plugin: nil
+    source: https://github.com/ros-navigation/nav2_minimal_turtlebot_simulation.git
```
Comment (Guillaumebeuzeboc) [inline turtlebot3c-model-snap/snapcraft.yaml:45]: maybe specifying a branch would be safer

Comment (artivis) [reply turtlebot3c-model-snap/snapcraft.yaml:45]: There is only one branch in this repo.

File: turtlebot3c-model-snap/snapcraft.yaml (hunk 1 above)
Comment (artivis) [reply turtlebot3c-model-snap/snapcraft.yaml:58]: I keep it for now as a reminder that other model would need this.

**Review by Guillaumebeuzeboc — APPROVED** [review]

**Pull request merged**